### DISCUSSION TITLE
Http code 400 error

### DISCUSSION THREAD

**Discussion by LecramRed** [bisand/node-red-contrib-tibber-api#81] [Q&A]
SOLVED
I get a 400 error when I try to connect to the Tibber API. It used to work, but not anymore. 
{"errors":[{"message":"Syntax Error: Unexpected Name \"This\".","extensions":{"code":"GRAPHQL_PARSE_FAILED"}}],"httpCode":400,"statusCode":400,"statusMessage":"Bad Request"}

What is going wrong? 

GraphQL Endpoint is https://api.tibber.com/v1-beta/gql
Token is correct (from the Tibber API explorer)

Update: I had another Cerbo which I connected to the same wifi and that worked well, while the first one still had issues. So I reverted to the normal firmware and reinstalled the Large version, and now it works fine. So there was apparently an issue with the software on the Cerbo.

**Comment by bisand** [comment]
Which function TibberQuery function are you calling? Can you post an example of the code? It's hard ti see what it could be without an example.

Thanks!

**Reply by LecramRed** [reply]
Thanks for you reply. It has already been solved. I had another Cerbo which I connected to the same wifi and that worked well, while the first one still had issues. So I reverted to the normal firmware and reinstalled the Large version, and now it works fine. So there was apparently an issue with the software on the Cerbo.
